lib/takeaway.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'order'
+
+class Takeaway
+  # Menu inspired by Dishoom. I recommend the Chicken Ruby
+  MENU = {
+    far_far:            2.5,
+    samosas:            4,
+    okra_fries:         4.2,
+    mattar_paneer:      6.5,
+    chicken_ruby:       8,
+    masala_prawns:      9,
+    biryani:            7.5,
+    naan:               2.5,
+    roti:               2.5,
+    basmati_rice:       2,
+    bhang_lassi:        3,
+    thums_up:           2.5,
+    kingfisher:         3.5,
+    chai:               2,
+    botal_water:        2
+  }
+
+  # IDEA: order_log and/or despatcher
+  def initialize(order_klass = ::Order)
+    @order_klass = order_klass
+    @order = nil
+  end
+
+  def view_menu
+    MENU.each do |k, v|
+      printf "%-15s %15s\n", "#{k.to_s.gsub(/_/, ' ').capitalize}:", "£#{v.to_s}"
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation. (https://github.com/bbatsov/ruby-style-guide#no-to-s)<br>Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)